src/Sulu/Bundle/MediaBundle/Media/Storage/LocalStorage.php

```diff
@@ -84,76 +76,48 @@ public function save($tempPath, $fileName, $version, $storageOption = null)
         }
         $this->filesystem->copy($tempPath, $filePath);
 
-        $this->addStorageOption('segment', $segment);
-        $this->addStorageOption('fileName', $fileName);
-
-        return json_encode($this->storageOption);
+        return [
```

Comment:
Somehow related to my `getPublicUrl` comment above. Wouldn't it be nice if the outside component simply retrieves the URL, instead of being bothered with an array, from which it has to know how to build the URL for it?

Reply:
could be different for each storage